packages/ember-data/tests/integration/adapter/serialize-test.js

```diff
@@ -0,0 +1,38 @@
+var run = Ember.run;
+var Person, env, store, adapter, serializer;
+
+module("integration/adapter/serialize - DS.Adapter integration test", {
+  setup: function() {
+    Person = DS.Model.extend({
+      name: DS.attr('string')
+    });
+
+    env = setupStore({ person: Person });
+    store = env.store;
+    adapter = env.adapter;
+    serializer = store.serializerFor('person');
+  },
+
+  teardown: function() {
+    run(env.container, 'destroy');
+  }
+});
+
+test("serialize() is delegated to the serializer", function() {
+  expect(1);
+
+  serializer.serialize = function(snapshot, options) {
+    deepEqual(options, { foo: 'bar' });
+    serializer._super(snapshot, options);
+  };
+
+  adapter.createRecord = function(store, type, snapshot) {
+    adapter.serialize(snapshot, { foo: 'bar' });
+    return Ember.RSVP.resolve({ id: 1 });
+  };
```

Comment:
I think it would be ok to just call `adapter.serialize` directly in the test instead of calling `store.createRecord` to invoke the `adapter.serialize`.